src/clients/c++/perf_client.cc

```diff
@@ -1590,12 +1600,17 @@ Report(
             << " infer/sec" << std::endl;
   if (percentile != -1) {
     std::cout << "    " << percentile
-              << "-th percentile latency: " << percentile_latency_us << " usec"
+              << "-th percentile latency: " << requested_percentile_latency_us << " usec"
```

Comment:
I would shorten "xx-th percentile" to "pxx" everywhere you print it. For example, "p99".